src/Microsoft.ML.FastTree/PACKAGE.md

```diff
@@ -0,0 +1,39 @@
+## About
+
+ML.NET is a cross-platform open-source machine learning framework which makes machine learning accessible to .NET developers.
+
```

Comment:
Usually the read me list the major types and small specific sample in the package readme. Are we using the same readme template used in the runtime repo? .

Reply:
So it looks like FastTree is the only one that I missed for the major types. I'll get those in. As far as samples go, we have a link to the code samples repo. Do you think thats enough or do you think we should actually put some samples in here? I basically just copied what had been done for the main ML.NET package to these others ones.